verilog/analysis/checkers/disable_fork_rule.h

```diff
@@ -0,0 +1,58 @@
+// Copyright 2017-2020 The Verible Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef VERIBLE_VERILOG_ANALYSIS_CHECKERS_DISABLE_STATEMENT_H_
+#define VERIBLE_VERILOG_ANALYSIS_CHECKERS_DISABLE_STATEMENT_H_
+
+#include <set>
+#include <string>
+
+#include "common/analysis/lint_rule_status.h"
+#include "common/analysis/syntax_tree_lint_rule.h"
+#include "common/text/symbol.h"
+#include "common/text/syntax_tree_context.h"
+#include "verilog/analysis/descriptions.h"
+
+namespace verilog {
+namespace analysis {
+
+// DisableForkNoLabelsRule finds occurrences of "dsiable fork_label"
```

Comment:
typo: `dsiable`

Reply:
I believe you are looking on the wrong git revision, I've removed this already.
Also, I changed the file names from disable_fork* to disable_non_sequential*,
but after your last review, I changed them to disable_statement* as suggested.
I believe this is the correct name for this rule.